src/utils/breach-logo.js

```diff
@@ -5,30 +5,63 @@
 /**
  * @param {object} breach
  * @param {Map<string, string>} logos Map of URLs to logos indexed by the domain name of the respective company
+ * @param {boolean} isEmail Is the icon being used in an email template?
```

Comment:
Suggestion, non-blocking: I think I'd do away with the conditionals, and just add the colour class-names regardless (with a comment that they're added for the emails), to simplify the code.

If you do want to keep `isEmail`, then I would at least recommend (like I'd do everywhere) to make all optional parameters named properties of an `options` parameter. That way, if we add more optional parameters in the future, we won't have to do `getBreachLogo(breach, logos, false, true, true, false)`, and it's easier to tell what the option does at the call site as well.

In other words:

```diff
- export function getBreachLogo (breach, logos, isEmail = false) {
+ export function getBreachLogo (breach, logos, options = {}) {
+   const isEmail = options.isEmail ?? false;
```

Reply:
I like both options. You are right though that another class does not hurt anyone and will reduce unnecessary complexity: [fd0a7d8](https://github.com/mozilla/blurts-server/pull/2960/commits/fd0a7d8b878eae5e438ceaa61924ff3e4e5e416a).